lms/djangoapps/onboarding/views.py

```diff
@@ -507,13 +507,13 @@ def admin_activation(request, activation_key):
 
     """
     context = {}
-    admin_activation = eval(request.GET.get('admin_activation', 'False'))
+    admin_activation = True if request.GET.get('admin_activation') == 'True' else False
 
     try:
         hash_key = OrganizationAdminHashKeys.objects.get(activation_hash=activation_key)
-        admin_change_confirmation = True if eval(request.GET.get('confirm', u'False')) else False
-        user_profile = UserExtendedProfile.objects.get(user__email=hash_key.suggested_admin_email)
-        user = user_profile.user
+        admin_change_confirmation = True if request.GET.get('confirm') == 'True' else False
```

Comment:
please execute the following expressions in python shell and see out out
bool('True')
bool('False')
bool(None)
bool('abc')

so if we simply write this as following it should work 
admin_change_confirmation = bool(request.GET.get('confirm'))

Reply:
bool('True') = True
bool('False') = True
bool(None) = False
bool('abc') = True

Reply:
bool('False') = True doesn't works correctly in our scenario.

Reply:
ok, please go for merge